tests/mapping_test_to_group.yaml

```diff
@@ -0,0 +1,106 @@
+platform_tests.test_cont_warm_reboot#test_continuous_reboot: L2
+radv.test_radv_ipv6_ra#test_radv_router_advertisement: L2
+radv.test_radv_ipv6_ra#test_solicited_router_advertisement: L2
+radv.test_radv_ipv6_ra#test_solicited_router_advertisement_with_m_flag: L2
+radv.test_radv_ipv6_ra#test_unsolicited_router_advertisement_with_m_flag: L2
+route.test_default_route#test_default_ipv6_route_next_hop_global_address: L2
+route.test_default_route#test_default_route_set_src: L2
+route.test_default_route#test_default_route_with_bgp_flap: L2
+route.test_route_flap#test_route_flap: L2
+route.test_route_perf#test_perf_add_remove_routes: L2
+route.test_static_route#test_static_route: L2
+route.test_static_route#test_static_route_ecmp: L2
+route.test_static_route#test_static_route_ecmp_ipv6: L2
+route.test_static_route#test_static_route_ipv6: L2
+scp.test_scp_copy#test_scp_copy: L2
+snmp.test_snmp_cpu#test_snmp_cpu: L2
+snmp.test_snmp_default_route#test_snmp_default_route: L2
+snmp.test_snmp_interfaces#test_snmp_interfaces: L2
+snmp.test_snmp_interfaces#test_snmp_interfaces_mibs: L2
+snmp.test_snmp_interfaces#test_snmp_mgmt_interface: L2
+snmp.test_snmp_lldp#test_snmp_lldp: L2
+snmp.test_snmp_loopback#test_snmp_loopback: L2
+snmp.test_snmp_pfc_counters#test_snmp_pfc_counters: L2
+snmp.test_snmp_queue#test_snmp_queues: L2
+ssh.test_ssh_ciphers#test_ssh_enc_ciphers: L2
+ssh.test_ssh_ciphers#test_ssh_kex: L2
+ssh.test_ssh_ciphers#test_ssh_macs: L2
+ssh.test_ssh_ciphers#test_ssh_protocol_version: L2
+ssh.test_ssh_limit#test_ssh_limits: L2
+ssh.test_ssh_stress#test_ssh_stress: L2
+tacacs.test_accounting#test_accounting_local_only: L2
+tacacs.test_accounting#test_accounting_tacacs_and_local: L2
+tacacs.test_accounting#test_accounting_tacacs_and_local_all_tacacs_server_down: L2
+tacacs.test_accounting#test_accounting_tacacs_only: L2
+tacacs.test_accounting#test_accounting_tacacs_only_all_tacacs_server_down: L2
+tacacs.test_accounting#test_accounting_tacacs_only_some_tacacs_server_down: L2
+tacacs.test_authorization#test_authorization_local: L2
+tacacs.test_authorization#test_authorization_tacacs_and_local: L2
+tacacs.test_authorization#test_authorization_tacacs_and_local_then_server_down_after_login: L2
+tacacs.test_authorization#test_authorization_tacacs_only: L2
+tacacs.test_authorization#test_authorization_tacacs_only_some_server_down: L2
+tacacs.test_authorization#test_authorization_tacacs_only_then_server_down_after_login: L2
+tacacs.test_authorization#test_backward_compatibility_disable_authorization: L2
+tacacs.test_authorization#test_bypass_authorization: L2
+tacacs.test_jit_user#test_jit_user: L2
+tacacs.test_ro_user#test_ro_user: L2
+tacacs.test_ro_user#test_ro_user_allowed_command: L2
+tacacs.test_ro_user#test_ro_user_banned_command: L2
+tacacs.test_ro_user#test_ro_user_ipv6: L2
+tacacs.test_rw_user#test_rw_user: L2
+tacacs.test_rw_user#test_rw_user_ipv6: L2
+test_hedgehog_smoke#test_bfd_smoke: L2
```

Comment:
Let's consider "smoke" tests according to geature. E.g.:
database, swss, syncd - "management"
bgp - L3
lldp - L2